app/models/blocked_edits_alert.rb

```diff
@@ -0,0 +1,9 @@
+class BlockedEditsAlert < Alert
+  def main_subject
+    "#{article.title}"
```

Comment:
There is no article for this alert; it gets created with only a user_id and target_user_id.